llvm/test/tools/llvm-readobj/ELF/Sparc/sparc64-elf-headers.test

```diff
@@ -0,0 +1,29 @@
+# RUN: yaml2obj %s -o %t -DFLAG_NAME=""
```

Comment:
Rather than split this into a separate test, I think it would be more normal to have these test cases in `sparc-elf-headers.test` too. You can use yaml2obj's --docnum option, if you can't sensibly share the YAML docs, although in this case, you could just parameterise the machine type via -D, like you do with the flag.